.ci/install_runtime.sh

```diff
@@ -110,13 +113,16 @@ if [ "$USE_VSOCK" == "yes" ]; then
 	echo "Configure use of VSOCK in ${runtime_config_path}"
 	sudo sed -i -e 's/^#use_vsock.*/use_vsock = true/' "${runtime_config_path}"
 
-	vsock_module="vhost_vsock"
-	echo "Check if ${vsock_module} is loaded"
-	if lsmod | grep -q "$vsock_module" &> /dev/null ; then
-		echo "Module ${vsock_module} is already loaded"
-	else
-		echo "Load ${vsock_module} module"
-		sudo modprobe "${vsock_module}"
+	# On OpenShift CI the vhost module should not be loaded on build time.
+	if [ "$OPENSHIFT_CI" == "false" ]; then
+		vsock_module="vhost_vsock"
+		echo "Check if ${vsock_module} is loaded"
+		if lsmod | grep -q "$vsock_module" &> /dev/null ; then
```

Comment:
It would be safer to anchor that string search with something like the following (you don't need the redirection I think?):

```bash
if lsmod | egrep -q "\<vhost_vsock\>" ; then
```

Also, there is a tacit assumption that a non-priv user can run `lsmod`, but I assume that's correct for this system?

Reply:
@jodh-intel updated this PR with a 3rd commit which address those comments. Is it now ready for merge?